app/Exceptions/Feedbacks/FeedbackNotCreatedException.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace App\Exceptions\Feedbacks;
+
+use Exception;
+
+class FeedbackNotCreatedException extends Exception
+{
+    //
```

Comment:
u can add default message in these custom exception classes as well. so that u can simply retain the message using `$exception->getMessage()`. Have a look over it.
If it adds value in yr scenario do add it